tools/mo/openvino/tools/mo/back/FakeOutputResolver.py

```diff
@@ -25,28 +25,21 @@ def find_and_replace_pattern(self, graph: Graph):
             for port in producer.out_ports().values():
                 if not port.disconnected():
                     producer_outputs += 1
-            if producer_outputs != 1:
-                # At this stage we don't know the type of output, so we rely on MO transformation which updates the
-                # Const type for elementwise operations in case of input data types mismatch
-                add = create_op_with_const_inputs(graph, Add, {1: int64_array(0)}, {'can_be_fused': False})
-                rename_nodes([(fake_output, name + '/TBD'), (add, name)])
-
-                prev_op_in_port = fake_output.in_port(0).get_connection().get_source()
-                # Get tensor names incoming to FakeOutput
-                tensor_names = prev_op_in_port.get_tensor_names()
-
-                # Remove tensor info from data node
-                prev_op_in_port.remove_tensor_names()
-
-                fake_output.in_port(0).get_connection().set_destination(add.in_port(0))
-                fake_output.out_port(0).get_connection().set_source(add.out_port(0))
-
-                # Move tensor names to Add op, which replaces FakeOutput
-                if len(tensor_names) > 0:
-                    add.out_port(0).add_tensor_names(tensor_names)
-
-            else:
-                result_in_port = fake_output.out_port(0).get_destination()
-                result_in_port.disconnect()
-                fake_output.in_port(0).get_connection().set_destination(result_in_port)
-                rename_nodes([(fake_output, name + '/TBD'), (producer, name)])
+            # At this stage we don't know the type of output, so we rely on MO transformation which updates the
+            # Const type for elementwise operations in case of input data types mismatch
+            add = create_op_with_const_inputs(graph, Add, {1: int64_array(0)}, {'can_be_fused': False})
```

Comment:
I understand that it is moved from the previous code. Can it have a type conflict with `int64`?

Reply:
I think there will appear warnings about it

Reply:
yes, we have warning about conversion int64 to float32 (for float output)